robot_skills/src/robot_skills/hero.py

```diff
@@ -50,6 +50,63 @@ def __init__(self, wait_services=False):
 
         self.configure()
 
+    def move_to_inspect_pose(self, inspect_target):
+        """
+        This poses the robot for an inspect.
+        :param inspection_position: kdl.Frame with the pose of the entity to be inspected.
+        """
+        # calculate the arm_lift_link which must be sent
+        z_over = 0.3  # height the robot should look over the surface
+        z_hh = 0.9  # height of the robots head at z_arm=0
+        torso_to_arm_ratio = 2  # motion of arm/motion of torso
```

Comment:
2.0? If you (accidentally) use two ints when you mean two floats you will get unexpected results (
```
In [13]: 1/2
Out[13]: 0
```